packages/gatsby/src/services/types.ts

```diff
@@ -16,6 +16,7 @@ export interface IMutationAction {
   payload: unknown[]
 }
 export interface IBuildContext {
+  firstRun?: boolean
```

Comment:
Should it be optional? Or mandatory and update all call sites to explicitly pass this on?